rust/operator-binary/src/spark_k8s_controller.rs

```diff
@@ -170,6 +175,16 @@ pub async fn reconcile(spark_application: Arc<SparkApplication>, ctx: Arc<Ctx>)
 
     let client = &ctx.client;
 
+    // Fix for #457
+    // Skip reconcyling the SparkApplication if it has a non empty status.
+    if spark_application.status.is_some() {
+        tracing::info!(
+            "Skip reconciling SparkApplication [{}] with non empty status",
+            spark_application.name_any()
+        );
```

Comment:
I *think* it's best practice to capture the variables using dedicated variables instead of including them in the text, but I would be interested in your opinion @NickLarsenNZ 
```suggestion
        tracing::info!(
            spark_application = spark_application.name_any(),
            "Skipped reconciling SparkApplication with non empty status"
        );
```